lara-typescript/src/interactive-api-shared/types.ts

```diff
@@ -91,6 +93,7 @@ export interface IAuthoringInitInteractive<AuthoredState = {}> {
 export interface IReportInitInteractive<InteractiveState = {}, AuthoredState = {}> {
   version: 1;
   mode: "report";
+  view?: string;
```

Comment:
@scytacki When I tried adding it only to `IReportInitInteractive` I would get errors like this when viewing the interactive in a dashboard.

![image](https://user-images.githubusercontent.com/367459/155595607-ac6a73d0-6e28-42ca-93dd-3040367a2519.png)

I wasn't sure how else to handle that. I thought about maybe somehow only adding the `view` property in line 120 of question-interactives' base-question-app.tsx when we know we're in a report, but I'm pretty sure it's already doing that. That line is inside the `renderReport` function.

Reply:
Maybe there'll have to be a different solution because when we restrict view to just `standalone` this error occurs:

<img width="1262" alt="image" src="https://user-images.githubusercontent.com/367459/155598185-a4081825-8262-4836-b532-0f7e7e8bd0d7.png">